po/fa.po

```diff
@@ -1,15 +1,18 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
+
```

Comment:
@javad-jafari, this blank line looks strange? Is this coming straight from Poedit?

Reply:
I removed the extra empty lines.